README.Rmd

```diff
@@ -16,16 +16,18 @@ Code to produce the [Supply Chain GHG Emission Factors](https://cfpub.epa.gov/si
 3. A local version of this repository, preferably a [release](https://github.com/USEPA/supply-chain-factors/releases) version to create EPA-verified factors.
 
 ### Optional Requirements
-1. [git](https://github.com/git-guides/install-git), if users wish to pull the code from this repository using git, instead of downloading a a zip file of the code.
-
-2. [Rstudio](https://www.rstudio.com/products/rstudio/download/#download), which provides a graphical interface and button to run this code (see Usage).
+1. [Git](https://github.com/git-guides/install-git), if users wish to clone the code from this repository using Git, instead of downloading a zip file of the code.
+2. [RStudio](https://www.rstudio.com/products/rstudio/download/#download), which provides a graphical interface and button to run this code (see Usage).
 
 ## Usage
 
 The code in this package can be used to generate the USEEIO-based Supply Chain GHG Emission Factors, based on options selected by the user at run-time. 
 
-These factors can be generated by [creating a project from the existing directory within Rstudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files in this repository. Once created, open
-the _CalculateEmissionFactors.Rmd_ in the editor, click on *Knit* and select *Knit with Parameters…*.
+If users choose to download a zip file of the code, they can unzip the file then [create a project from the existing directory within RStudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files.
+
+If users choose to clone this repository, they can [create a project by cloing the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```

Comment:
I think its should rather be

"If users have git installed, the users can create a project by cloning the repository with RStudio"

Reply:
```suggestion
If users have git installed, they can [create a project by cloning the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```